docs/installation.md

```diff
@@ -1,132 +1,208 @@
+And `<path>` is the desired install path. `-DCMAKE_INSTALL_PREFIX=<path>` is optional, if it is specified manually (not recommended), keep note of this path as we will need it later.
+
+
+> Window supports is experimental. This means using MSVC may work, however this is currently untested. The [cpp compiler support]() page seems to imply that MSVC 19.30 or newer is required to compile `jluna`.
+
+Some errors may appear here, if this is the case, head to [troubleshooting](#troubleshooting).
+
+If the command does not recognize the compiler, even though you are sure it is installed, it may be necessary to specify the full path to the compiler executable instead, like so:
 
-```julia
-println(joinpath(Sys.BINDIR, ".."))
-```
 ```
-/path/to/your/julia/bin/..
+-DCMAKE_CXX_COMPILER=/usr/bin/g++-10
 ```
 
-And copy-paste the resulting output to assign `JULIA_PATH` in bash like so:
+### Make Install
+
+Having successfully configured cmake, we now call:
 
 ```bash
-export JULIA_PATH=/path/to/your/julia/bin/..
+# in Desktop/jluna/build
+make install
 ```
 
-We can now continue to compiling `jluna` using cmake, being sure to stay in the same bash session where we set `JULIA_PATH`. To set `JULIA_PATH` globally, consider consulting [this guide](https://unix.stackexchange.com/questions/117467/how-to-permanently-set-environmental-variables). This way, it does not have to re-set anytime a bash session ends.
+Which will create two shared libraries `libjluna.*`, and `libjluna_c_adapter.*` in the specified install directories, or the default install directory if unspecified. `*`, here, is platform dependent, usually `.so` on unix and `.lib` or `.dll` on windows. 
+
+Now that `jluna` is installed on our system, we can link any executable or library inside any non-`jluna` `CMakeLists.txt` using:
 
-### Building `jluna`
+```cmake
+# in users own CMakeLists.txt
+find_library(jluna REQUIRED 
+    NAMES jluna
+)
+```
 
-With `JULIA_PATH` set correctly, we navigate into `~/my_project/jluna/` and create a new build directory, then call cmake from within that directory:
+If a custom install directory was specified, we need to make cmake aware of this:
 
-```bash
-cd ~/my_project/jluna
-mkdir build
-cd build
-cmake -D CMAKE_CXX_COMPILER=g++-11 .. # or clang++-12
-make
+```cmake
+# in users own CMakeLists.txt
+find_library(jluna REQUIRED 
+    NAMES jluna
+    PATHS <install path>
+)
 ```
 
-You may need to specify the absolute path to `g++-11` / `clang-12` if their respective executables are not available in the default search paths.
+Where `<install path>` is the path specified as `-DCMAKE_INSTALL_PREFIX` during configuration before. 
 
-When compiling, warnings of the following type may appear:
+If `find_library` is still not able to find `jluna`, we may need to manually specify the shared library suffixes like so:
 
+```cpp
+set(CMAKE_FIND_LIBRARY_SUFFIXES_WIN32 ".lib;.dll;.dll.a;")
+set(CMAKE_FIND_LIBRARY_SUFFIXES_UNIX ".so")
```

Comment:
This is not relevant here and the reason why this kind of code was included in the FindJulia.cmake file was explained in the PR.